libs/utility/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 add_library(Logger STATIC)
 target_sources(Logger PRIVATE src/Logger.cpp)
 target_include_directories(Logger PUBLIC include)
-target_set_mbed_dependency(Logger)
+target_link_libraries(Logger PUBLIC mbed-os)
 
 add_library(LookupTable INTERFACE)
 target_include_directories(LookupTable INTERFACE include)
 
-add_library(units INTERFACE)
-target_include_directories(units INTERFACE include)
\ No newline at end of file
+add_library(Units INTERFACE)
```

Comment:
we really need to standardize target names. so far I've been using all Pascal case, but we can switch to just dashed to match mbed... but for now its all pascal so we should be telling people that in CR